infinite_tracing/lib/infinite_tracing/record_status_handler.rb

```diff
@@ -46,6 +26,30 @@ def stop
           @worker = nil
         end
       end
+
+      private
+
+      def build_worker
```

Comment:
We might want to rename this method, since it's not exactly building things. this is the loop that handles all of the responses from the trace observer, maybe like `handle_response` or `run_handler` or `process_responses`, or something like that maybe?